packages/react/src/status-badge/StatusBadge.tsx

```diff
@@ -57,7 +57,8 @@ export const StatusBadge = ({
 					borderRadius,
 					'& svg': {
 						flexShrink: 0,
-						width: iconWidth,
+						height: iconSize,
```

Comment:
This is the legacy version of the component, which had a height that didn't match the width